homeassistant/components/saj/sensor.py

```diff
@@ -48,139 +58,161 @@
     {
         vol.Required(CONF_HOST): cv.string,
         vol.Optional(CONF_NAME): cv.string,
+        vol.Optional(CONF_SCAN_INTERVAL): cv.positive_int,
```

Comment:
We no longer accept changes to the YAML configuration schema. Please revert this change.